### ISSUE TITLE
Multiple pickups from a station print multiple times.

### ISSUE THREAD

**Issue by CobiELF** [CobiELF/SAD-planner#2]
e.g.

```
Pickup 2 package(s) on crusader.
Pickup 2 package(s) on crusader.
```
Should print only once.

```
Pickup 2 package(s) on arccorp.
Drop off package(s) on arccorp (sourced from crusader).
Pickup 2 package(s) on arccorp.
```
should provide a hint as to what's going on.

**Comment by CobiELF** [comment]
A possible solution is to move the pickup loop to under the previous destination check.

**Comment by CobiELF** [comment]
fixed now 4b8c60512356f6f1ba1262742ca8b512d7a7681b
